docs/engineering-defaults.md

```diff
@@ -1,7 +1,147 @@
 # Engineering Defaults
 
+Much of OpenO11y's guidance is based on or enhanced by these best practices for
+engineering teams.
+
 ## Pair Programming
 
-## Trunk Based Development
+Pair programming is a software development technique in which two developers work
+together at one workstation. One, the driver, writes code while the other, the
+observer or navigator, reviews each line of code as it is typed in. The two
+developers switch roles frequently.
+
+This method provides extremely fast feedback to developers about their code
+changes and design decisions before. In turn, this prevents wasted effort and produces
+cleaner, simpler code at a faster rate.
+
+Tooling can greatly improve the developers' experience while pair programming.
+Audio calls and screen sharing enable pair programming for engineers in
+different physical locations. Similarly, IDE tools such as Visual Studio Code's
+[Live Share][liveshare] extension allow both parties to develop code changes
+simultaneously in a single workspace.
 
 ## Small Batch Delivery
+
+OpenO11y encourages teams to deliver in small batches. Delivering small portions
+of work enables teams to deploy, demo, and request feedback more frequently.
+Much like pair programming, this practice aids teams in coordinating their
+efforts and safeguards against individuals investing time and energy in
+developing an incorrect solution.
+
+## Trunk Based Development
```

Comment:
I'd like to see more of the original content in this one.  It has resonated pretty well with our customers.

Reply:
Let's iterate on this with additional PRs